Kinvey-Xamarin-Android/Auth/UserExtension.cs

```diff
@@ -13,17 +13,25 @@
 
 using Kinvey;
 using Android.Content;
+using System;
 
 namespace Kinvey
 {
 	public static class UserExtension
 	{
-		public static void OnOAuthCallbackRecieved(this User user, Intent intent)
+        [Obsolete("This method has been deprecated. Please use OnOAuthCallbackReceived() instead.")]
+        public static void OnOAuthCallbackRecieved(this User user, Intent intent)
 		{
 			global::Android.Net.Uri uri = intent.Data;
 			string accessToken = uri.GetQueryParameter("code");
 			User.GetMICAccessTokenAsync(accessToken);
 		}
 
-	}
+        public static void OnOAuthCallbackReceived(this User user, Intent intent)
```

Comment:
@romanogolikhin Instead of copying the implementation over to a new method, can we have the old, obsolete method just call the new method?

Reply:
Ok.

Reply:
@vinaygahlawat I made such way as I thought we would remove all [obsolete] methods in future.
Is it correct?